protobuf/types/service.proto

```diff
@@ -36,6 +36,17 @@ message Service {
 
   // Task is a function that requires inputs and returns output.
   message Task {
+    message Price {
+      string perCall = 1 [
+        (gogoproto.moretags) = 'hash:"name:1" validate:"bigint"'
+      ];
+      string perSec = 2 [
+        (gogoproto.moretags) = 'hash:"name:2" validate:"bigint"'
+      ];
+      string perKB = 3 [
+        (gogoproto.moretags) = 'hash:"name:3" validate:"bigint"'
+      ];
```

Comment:
I'm not against the string type in proto so it's easier to implement in other languages,
But in go it should be `Int`. Did you try `gogoproto.customtype` or `gogoproto.casttype`?

Reply:
Is `bigint` validation accept empty?
I suggest to explicitly said if it requires or not:
```suggestion
    message Price {
      string perCall = 1 [
        (gogoproto.moretags) = 'hash:"name:1" validate:"required,bigint"'
      ];
      string perSec = 2 [
        (gogoproto.moretags) = 'hash:"name:2" validate:"required,bigint"'
      ];
      string perKB = 3 [
        (gogoproto.moretags) = 'hash:"name:3" validate:"required,bigint"'
      ];
```
or if optional:
```suggestion
    message Price {
      string perCall = 1 [
        (gogoproto.moretags) = 'hash:"name:1" validate:"omitempty,bigint"'
      ];
      string perSec = 2 [
        (gogoproto.moretags) = 'hash:"name:2" validate:"omitempty,bigint"'
      ];
      string perKB = 3 [
        (gogoproto.moretags) = 'hash:"name:3" validate:"omitempty,bigint"'
      ];
```

Reply:
see https://github.com/mesg-foundation/engine/pull/1842#discussion_r432923254

Reply:
done